### PR TITLE
SAR-8981 Update content on details not found page

### PR DESCRIPTION
**Pull request by woollett-r**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -111,10 +111,15 @@ check-your-answers.no_tax-identifiers                       = I do not want to p
 ## Details Not Found Page
 details-not-found-page.title                                = The details you entered did not match our records
 details-not-found-page.heading                              = The details you entered did not match our records
-details-not-found-page.line_1                               = You cannot continue to register using this service. You need to
-details-not-found-page.link_1                               = contact the National Insurance team (opens in a new tab)
-details-not-found-page.line_2                               = If you entered the wrong details, you can
+details-not-found-page.line_1                               = We could not match the details you entered with records held by HMRC.
+details-not-found-page.line_2                               = If you used the wrong details, you can
 details-not-found-page.link_2                               = try again using different details.
+details-not-found-page.line_3                               = If you used the correct details, you cannot continue to register using this online service.
```
Comment (refaque-ahmad) [inline conf/messages:117]: Can we confirmed it's "entered" or "used"

Comment (woollett-r) [reply conf/messages:117]: Liz's advice is to use "used" for now.

File: conf/messages [hunk 2]
```diff
@@ -111,10 +111,15 @@ check-your-answers.no_tax-identifiers                       = I do not want to p
...
+details-not-found-page.line_4.1                             = You need to
+details-not-found-page.link_4                               = contact the National Insurance team (opens in a new tab)
+details-not-found-page.line_4.2                             = and
+details-not-found-page.line_4.3                             = tell them there is an issue with your National Insurance number,
+details-not-found-page.line_4.4                             = no matter what tax regime you''re using.
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply conf/messages:122]: Actually no. In the messages file you need to use a double apostrophe for an apostrophe to appear on the web page. I'm not sure why, but I have come across this before.

File: it/uk/gov/hmrc/soletraderidentificationfrontend/assets/MessageLookup.scala [hunk 3]
```diff
@@ -171,10 +171,12 @@ object MessageLookup {
   object DetailsNotFound {
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply it/uk/gov/hmrc/soletraderidentificationfrontend/assets/MessageLookup.scala:171]: Done

File: it/uk/gov/hmrc/soletraderidentificationfrontend/views/DetailsNotFoundViewTests.scala [hunk 4]
```diff
@@ -12,6 +27,8 @@ import uk.gov.hmrc.soletraderidentificationfrontend.utils.ViewSpecHelper.Element
 trait DetailsNotFoundViewTests {
   this: ComponentSpecHelper =>
 
+  val four: Int = 4
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply it/uk/gov/hmrc/soletraderidentificationfrontend/views/DetailsNotFoundViewTests.scala:30]: Yes, Intellij considers 4 to be a magic number so I thought I'd create a val to keep it quiet.

**Review by refaque-ahmad — APPROVED** [review]

**Pull request merged**